css/style.css

```diff
@@ -0,0 +1,682 @@
+    padding-left: 44px;
+    max-width: 356px;
+  }
+
+  .main__header {
+    font-size: 50px;
+    line-height: 48px;
+    margin-bottom: 44px;
+  }
+
+  .main__link {
+    font-size: 20px;
+    line-height: 19px;
+  }
+
+  .main__link::after {
+    left: 130px;
+    bottom: 3px;
+    background: url(../icons/Arrow-big.png);
+    width: 73px;
+    height: 16px;
+  }
+
+  .main__subheader {
+    font-size: 30px;
+    line-height: 29px;
+    margin-bottom: 32px;
+    padding-right: 15px;
+  }
+
+  .main__card-wrapper {
+    padding: 0 50px;
+  }
+
+  .product-card {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-width: 304px;
+  }
+
+  .product-card__img {
+    height: 215px;
+    padding: 0px 58px 34px 79px;
+  }
+
+  .product-card:nth-child(3),
+  .product-card:nth-child(4) {
+    display: none;
+  }
+
+  .product-card::after {
+    display: none;
+  }
+
+  .product-card__options {
+    width: 341px;
+    justify-content: space-around;
+  }
+
+  .product-card__discr {
+    font-size: 20px;
+    line-height: 19px;
+    padding-bottom: 17px;
+  }
+
+  .product-card__price {
+    font-size: 25px;
+    line-height: 24px;
+    padding-left: 8px;
+  }
+
+  .product-card__sel {
+    width: 94px;
+    padding-right: 0px;
+  }
+
+  .product-card__btn {
+    height: 20px;
+    width: 20px;
+  }
+}
+
+@media (max-width: 375px) {
+  body {
+    min-width: 375px;
+  }
+
+  .heading {
+    min-height: 72px;
+  }
+
+  .heading__hamburger-wrapper {
+    padding: 19px 44px;
+  }
+
+  .heading__hamburger-logo {
+    width: 104px;
+  }
+
+  .heading__hamburger span {
+    height: 5px;
+    width: 41px;
+  }
+
+  .heading__hamburger:hover span:nth-child(1) {
+    transform: translateY(13px) rotate(-45deg);
+  }
+
+  .heading__hamburger:hover span:nth-child(2) {
+    transform: translateY(2px) rotate(45deg);
+  }
+
+  .main__wrapper {
+    height: 191px;
+    background: url(../img/banner-bg-mobile.png);
+    margin-bottom: 36px;
+  }
+
+  .main__text-wrapper {
+    padding-top: 65px;
+    padding-left: 48px;
+    max-width: 206px;
+  }
+
+  .main__header {
+    font-size: 20px;
+    line-height: 19px;
+    margin-bottom: 9px;
+  }
+
+  .main__link {
+    font-size: 12px;
+    line-height: 11px;
+  }
+
+  .main__link::after {
+    left: 77px;
+    bottom: 1px;
+    background: url(../icons/arrow.png);
+    width: 44px;
+    height: 8px;
+  }
+
+  .main__subheader {
+    font-size: 20px;
+    line-height: 19px;
+    margin-bottom: 4px;
+    padding-left: 4px;
+  }
+
+  .main__card-wrapper {
+    padding: 0 46px;
+  }
+
+  .product-card {
+    min-width: 268px;
+  }
+
+  .product-card__img {
+    height: 151px;
+    min-width: 163px;
+    padding-bottom: 27px;
+  }
+
+  .product-card:nth-child(2) {
+    display: none;
+  }
+
+  .product-card__discr {
+    width: 100%;
+    font-size: 15px;
+    line-height: 14px;
+    padding-left: 30px;
+    padding-bottom: 6px;
+  }
+
+  .product-card__options {
+/*    width: 100%;*/
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5575166)